### ISSUE TITLE
[BUG]: recently added support for `"new"` headless mode fails with Playwright

### ISSUE THREAD

**Issue by dgrebb** [garris/BackstopJS#1534]
## Summary

As documented in [the README](https://github.com/dgrebb/BackstopJS#setting-puppeteer-and-playwright-option-flags), setting `engineOptions.headless = "new"` is passed to Playwright as expected, but fails.

### `backstopjs@6.3.2`

```shell
Creating Browser
No Playwright browser specified, assuming Chromium.
      COMMAND | Command "test" ended with an error after [0.003s]
      COMMAND | browserType.launch: headless: expected boolean, got string
```

## Details

Playwright supports `"new"` headless mode, set as a string, but only by defining `ignoredDefaultArgs: ["--headless"]`.

This isn't well documented, but one can see Playwright's [`headless` argument](https://playwright.dev/docs/api/class-browsertype#browser-type-launch-option-headless) expects a boolean.

Deeply burred in [a GitHub issue](https://github.com/microsoft/playwright/issues/21194#issuecomment-1444276676), there is a small example of how to make `"new"` headless mode work:

```javascript
  projects: [
    {
      name: 'chromium',
      use: {
        launchOptions: {
          ignoreDefaultArgs: ['--headless'],
          args: ['--headless=new']
        }
      },
    },
```

(#1525) (dgrebb/backstopjs#73)

**Comment by dgrebb** [comment]
Found and fixing 👍

**Comment by dgrebb** [comment]
Closed with #1535
